src/commands/auth/login.ts

```diff
@@ -6,17 +6,51 @@ import { askCredentials } from "./util";
 import { cliContainer } from "../../container";
 
 async function run(argv: Arguments) {
-  const answers = await askCredentials(argv);
+  let password: string | undefined;
+  if (argv["password-stdin"]) {
+    password = await readStdin();
```

Comment:
We should require `email` to be provided via parameters, since stdin would not be available anymore.